mapstruct-2/src/main/java/com/baeldung/mapstruct/date/model/UserDTO.java

```diff
@@ -0,0 +1,24 @@
+package com.baeldung.mapstruct.date.model;
+
+public class UserDTO {
```

Comment:
We follow https://google.github.io/styleguide/javaguide.html
```suggestion
public class UserDto {
```

(same elsewhere, DTO -> Dto)

Reply:
Will fix later